test/export/InstanceExporter.test.ts

```diff
@@ -4424,6 +4425,132 @@ describe('InstanceExporter', () => {
+        const activityAssignmentRule = new AssignmentRule('category[+]');
+        activityAssignmentRule.value = new FshCode(
+          'activity',
+          'http://terminology.hl7.org/CodeSystem/observation-category',
+          'Activity'
+        );
+        const statusAssignmentRule = new AssignmentRule('status');
+        statusAssignmentRule.value = new FshCode('final');
+        const codeAssignmentRule = new AssignmentRule('code');
+        codeAssignmentRule.value = new FshCode('wow');
+        labInstance.rules.push(
+          labPathRule,
+          activityAssignmentRule,
+          statusAssignmentRule,
+          codeAssignmentRule
+        );
+
+        const exported = exportInstance(labInstance);
+        // With manualSliceOrdering = true, PathRule keeps lab slice at first element
+        // in array and category[+] adds activity element after
+        expect(exported.category).toHaveLength(2);
+        expect(exported.category[0]).toEqual({
+          coding: [
+            {
+              code: 'laboratory',
+              system: 'http://terminology.hl7.org/CodeSystem/observation-category'
+            }
+          ]
+        });
+        expect(exported.category[1]).toEqual({
+          coding: [
+            {
+              code: 'activity',
+              system: 'http://terminology.hl7.org/CodeSystem/observation-category',
+              display: 'Activity'
+            }
+          ]
+        });
+        expect(loggerSpy.getAllMessages('error')).toHaveLength(0);
+        expect(loggerSpy.getAllMessages('warn')).toHaveLength(0);
+      });
+
+      it('should not add null values with path rules', () => {
+        // Instance: ExampleDS
+        // InstanceOf: DiagnosticReport
+        // * presentedForm
+        const diagnosticReport = new Instance('ExampleDS');
+        diagnosticReport.instanceOf = 'DiagnosticReport';
+        const pathRule = new PathRule('presentedForm');
+        diagnosticReport.rules.push(pathRule);
+
+        const exportedInstance = exportInstance(diagnosticReport);
+        expect(exportedInstance.presentedForm).toBeUndefined();
+      });
+
+      it('should add an entry for each index used in a path rule', () => {
+        const observation = new Profile('ObsWithInterpretation');
+        observation.parent = 'Observation';
+        const cardRule = new CardRule('interpretation.text');
+        cardRule.min = 1;
+        const assignmentRule = new AssignmentRule('interpretation.text');
+        assignmentRule.value = 'Important interpretations here';
+        observation.rules.push(cardRule, assignmentRule);
+        doc.profiles.set(observation.name, observation);
+
+        const instance = new Instance('MyObsWithInterpretation');
+        instance.instanceOf = 'ObsWithInterpretation';
+        const pathRuleIndex0 = new PathRule('interpretation[0]');
+        const pathRuleIndex1 = new PathRule('interpretation[1]');
+        instance.rules.push(pathRuleIndex0, pathRuleIndex1);
```

Comment:
Can you add the FSH comments for these ones? I find that helpful to more quickly understand the test.

Reply:
Done in [0e78678](https://github.com/FHIR/sushi/pull/1252/commits/0e78678cd1c2c888cae6437681de474c250a9831).